src/common/components/notifications/index.tsx

```diff
@@ -48,6 +48,8 @@ export const date2key = (s: string): string => {
   return s;
 };
 
+const buttonStatus: string[] = ["All", "Unread", "Read"];
```

Comment:
Hi @feruzm 
I made an enum and then used it. I want to share one more thing. I am thinking to show count of notifications just like in the image. What's your suggestion on this? If you like it, then I will start working on it.These are just hard coded values.
![image](https://user-images.githubusercontent.com/106739598/233002804-aa72f91f-ace0-49b9-9502-fe3c1db99fe3.png)

Reply:
Hi @feruzm . I have implemented this. Kindly Review this video and let me know if you feel any suggestion. If everything is fine then please let me know I will push the updated code. 
Thanks!

https://user-images.githubusercontent.com/106739598/233025089-fe4646c9-3b06-4e5b-a107-47688cc6a07d.mp4

Reply:
> @MTSaeed looks like good idea, but we don't always fetch full notifications for an account, there is pagination and I am afraid it will cause more confusion thinking

In order to avoid from any confusion, we can add + symbol after All's count. But when there is no more notification, we can just show the total count.
The same thing we have implemented for the proposals.
Is it make sense?